tests/Zend/Form/Decorator/LabelTest.php

```diff
@@ -283,7 +283,7 @@ public function testSettingTagToEmptyValueShouldSetTagToNull()
                         ->setOptions(array('tag' => 'dt'));
         $this->decorator->setTag('');
         $tag = $this->decorator->getTag();
-        $this->assertTrue(null === $tag, $tag);
```

Comment:
`null` isn't allowed as second parameter (needs to be a string), and it's not particularly useful here since it's a label for the failure to be output when test fails.  Changed to `assertNull`.